### PR TITLE
Syntax Diagram in Syntax Visualizer : change background color of skipped token nodes for accessibility

### PR DESCRIPTION
**Pull request by chborl**
Fixes https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_workitems/edit/483449

Customer scenario
In Syntax Visualizer, when viewing a Syntax Diagram that included a skipped token, the node was red with white font.  This has a contrast ratio of 4.0:1 which does not meet the minimum accessibility requirement of 4.5:1. After changing the background to maroon is has a contrast ratio of 10.9:1

Bugs this fixes
https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_workitems/edit/483449

Workarounds, if any
no workaround

Risk
No risk, color change only

Performance impact
None

Is this a regression from a previous update?
No

Root cause analysis
High Contrast settings were not tested initially

How was the bug found?
Accessibilty testers

### CODE REVIEW

**Comment by chborl** [comment]
@dotnet/roslyn-analysis @dotnet/roslyn-ide for review

File: src/Tools/SyntaxVisualizer/Roslyn.SyntaxVisualizer.DgmlHelper/SyntaxDgmlHelper.vb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```
Comment (333fred) [inline src/Tools/SyntaxVisualizer/Roslyn.SyntaxVisualizer.DgmlHelper/SyntaxDgmlHelper.vb]: Looks like the BOM might have changed? Can you make sure that this part of the file isn't touched.

**Comment by jinujoseph** [comment]
@Pilchie for ask mode approval (Accessibility bug)

**Comment by Pilchie** [comment]
For what release?  15.6 is now in escrow, and would need @MattGertz's approval.

**Comment by jinujoseph** [comment]
this is for 15.6 , was thinking of getting approval on insertion PR from @MattGertz with all fixes

**Comment by jinujoseph** [comment]
[pending shiproom approval]

**Comment by jinujoseph** [comment]
Approved to merge via [link](https://devdiv.visualstudio.com/DevDiv/NET%20Developer%20Experience%20Productivity/_workitems/edit/483449)

**Pull request merged**